### PR TITLE
chore(CS): Update central server related system paramater documentation 

### PR DESCRIPTION
**Pull request by ovidijusnortal**

### CODE REVIEW

**Comment by ovidijusnortal** [comment]
[rerun-ci]

**Comment by ovidijusnortal** [comment]
[rerun-ci]

File: doc/Manuals/ug-syspar_x-road_v6_system_parameters.md [hunk 1]
```diff
@@ -484,14 +485,20 @@ For instructions on how to change the parameter values, see section [Changing th
...
+| allowed-hostnames                 |                                                     | Determines which hostnames are allowed to be used in "Host" header of the HTTP requests. Any hostname is allowed when left unspecified.                                                                                                                                                                                                                                                                                                                                                                                                                        |
```
Comment (petkivim) [inline doc/Manuals/ug-syspar_x-road_v6_system_parameters.md:490]: If multiple hostnames are specified, how different hostnames should be separated from one another? Is it a comma separated list or something else?

File: doc/Manuals/ug-syspar_x-road_v6_system_parameters.md [hunk 2]
```diff
@@ -484,14 +485,20 @@ For instructions on how to change the parameter values, see section [Changing th
...
+| rate-limit-enabled                | true <br/> _(see Description for more information)_ | Controls whether the built-in rate limiting is enabled. <br/> Note. If the service is behind a reverse proxy (default), the proxy needs to forward the real IP address for the rate-limiting to work correctly. Therefore, by default, using forward headers is enabled. <br/> If the service is exposed directly, it must not use forwarded (e.g. X-Forwarded-For) headers (can be spoofed by clients), and the corresponding configuration (server.forward-headers-strategy) needs to be disabled(See [The configuration file](#61-the-configuration-file)). |
```
Comment (petkivim) [inline doc/Manuals/ug-syspar_x-road_v6_system_parameters.md:491]: This description could be updated a little bit:

> Controls whether the built-in rate limiting is enabled. <br/> Note. If the service is behind a reverse proxy (default), the proxy needs to forward the real IP address for the rate-limiting to work correctly. Therefore, by default, using forward headers is enabled. <br/> If the service is exposed directly, it must not use forwarded (e.g. `X-Forwarded-For`) headers (can be spoofed by clients), and the corresponding configuration (`server.forward-headers-strategy`) needs to be disabled using command line arguments (see [additional information](#61-the-configuration-file)).

File: doc/Manuals/ug-syspar_x-road_v6_system_parameters.md [hunk 3]
```diff
@@ -484,14 +485,20 @@ For instructions on how to change the parameter values, see section [Changing th
...
+| backup-allowed-content-types      | application/pgp-encrypted                           | Determines which file content types are allowed when uploading backup file. Content type is deducted from file extension and file contents. To disable this check, leave this value empty.                                                                                                                                                                                                                                                                                                                                                                     |
```
Comment (petkivim) [inline doc/Manuals/ug-syspar_x-road_v6_system_parameters.md:502]: Should `rateLimitExpireAfterAccessMinutes` be mentioned too?

File: doc/Manuals/ug-syspar_x-road_v6_system_parameters.md [hunk 4]
```diff
@@ -503,42 +510,42 @@ For instructions on how to change the parameter values, see section [Changing th
...
+| rate-limit-enabled             | true <br/> _(see Description for more information)_ | Controls whether the built-in rate limiting is enabled. <br/> Note. If the service is behind a reverse proxy (default), the proxy needs to forward the real IP address for the rate-limiting to work correctly. Therefore, by default, using forward headers is enabled. <br/> If the service is exposed directly, it must not use forwarded (e.g. X-Forwarded-For) headers (can be spoofed by clients), and the corresponding configuration (server.forward-headers-strategy) needs to be disabled(See [The configuration file](#61-the-configuration-file)). |
```
Comment (petkivim) [inline doc/Manuals/ug-syspar_x-road_v6_system_parameters.md:515]: This description could be updated a little bit:

> Controls whether the built-in rate limiting is enabled. <br/> Note. If the service is behind a reverse proxy (default), the proxy needs to forward the real IP address for the rate-limiting to work correctly. Therefore, by default, using forward headers is enabled. <br/> If the service is exposed directly, it must not use forwarded (e.g. `X-Forwarded-For`) headers (can be spoofed by clients), and the corresponding configuration (`server.forward-headers-strategy`) needs to be disabled using command line arguments (see [additional information](#61-the-configuration-file)).

File: doc/Manuals/ug-syspar_x-road_v6_system_parameters.md [hunk 5]
```diff
@@ -503,42 +510,42 @@ For instructions on how to change the parameter values, see section [Changing th
...
```
Comment (petkivim) [inline doc/Manuals/ug-syspar_x-road_v6_system_parameters.md:528]: This description could be updated a little bit:

> Controls whether the built-in rate limiting is enabled. <br/> Note. If the service is behind a reverse proxy (default), the proxy needs to forward the real IP address for the rate-limiting to work correctly. Therefore, by default, using forward headers is enabled. <br/> If the service is exposed directly, it must not use forwarded (e.g. `X-Forwarded-For`) headers (can be spoofed by clients), and the corresponding configuration (`server.forward-headers-strategy`) needs to be disabled using command line arguments (see [additional information](#61-the-configuration-file)).

File: doc/Manuals/ug-syspar_x-road_v6_system_parameters.md [hunk 6]
```diff
@@ -503,42 +510,42 @@ For instructions on how to change the parameter values, see section [Changing th
...
+| api-token                      | <a token with MANAGEMENT_SERVICE role>              | API token for the central server management API (required). The token needs to have the MANAGEMENT_SERVICE role (and for security, no other roles).                                                                                                                                                                                                                                                                                                                                                                                                            |
```
Comment (petkivim) [inline doc/Manuals/ug-syspar_x-road_v6_system_parameters.md:523]: Should `rateLimitExpireAfterAccessMinutes` be mentioned too?

File: doc/Manuals/ug-syspar_x-road_v6_system_parameters.md [hunk 7]
```diff
@@ -503,42 +510,42 @@ For instructions on how to change the parameter values, see section [Changing th
...
```
Comment (petkivim) [inline doc/Manuals/ug-syspar_x-road_v6_system_parameters.md:536]: Should `rateLimitExpireAfterAccessMinutes` be mentioned too?

File: doc/Manuals/ug-syspar_x-road_v6_system_parameters.md [hunk 8]
```diff
@@ -81,6 +81,7 @@ Doc. ID: UG-SYSPAR
...
 | 26.05.2023 | 2.70    | Add rate-limit properties                                                                                                                                                                                                                                                                                                                                                                                      | Ričardas Bučiūnas          |
+| 01.06.2023 | 2.71    | Update and add missing descriptions for to Central server configuration parameters.                                                                                                                                                                                                                                                                                                                            | Ovidijus Narkevičius       |
```
Comment (petkivim) [inline doc/Manuals/ug-syspar_x-road_v6_system_parameters.md:84]: The version number in the beginning of the document should be updated from 2.70 to 2.71.

**Review by petkivim — APPROVED** [review]
Some minor changes to the System Parameters document are needed.

**Comment by ovidijusnortal** [comment]
[rerun-ci]

**Pull request merged**